src/screen-components/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -608,7 +608,20 @@ function EstimatedCallRow({
         onPress={() => handleQuayPress(call.quay)}
         testID={'legType_' + group}
       >
-        <ThemeText testID="quayName">{getQuayName(call.quay)}</ThemeText>
+        <ThemeText testID="quayName">
+          {getQuayName(call.quay)}
+          {isStartOfGroup && !!call.quay.description && (
+            <ThemeText
```

Comment:
Don't think it shouldn't be necessary to nest them here either. The texts below are at least read out as one screen reader focus area. Maybe use AccessibleText with `pause="before"` like the stuff below.

Reply:
Or will that put the text below the stop marker line thingy? 🤔